spec/spec_helper.cr

```diff
@@ -8,6 +8,7 @@ include RoutesHelper
 Pulsar.enable_test_mode!
 
 Lucky::AssetHelpers.load_manifest
+Lucky::AssetHelpers.load_manifest("./public/vite-manifest.json", use_vite: true)
```

Comment:
In practice you would never run this more than once, but this was the only way I could test it since it happens at compile-time.